src/persist/updatableFileAdapter.ts

```diff
@@ -0,0 +1,22 @@
+import { FileAdapter } from './fileAdapter';
+import { UpdatableAdapter } from './updatableAdapter';
+
+/**
+ * UpdatableFileAdapter is the file adapter for Casbin.
+ * It can updates policy from file.
+ */
+export class UpdatableFileAdapter extends FileAdapter implements UpdatableAdapter {
```

Comment:
FileAdapter is a class name, not an interface name. FileAdapter will always be called FileAdapter, no matter how many interfaces it has implemented. I have no idea why the names like `UpdatableFileAdapter` came out.

Reply:
This file is inspired by BatchFileAdapter, maybe this is an example?

Reply:
I think first we need correctify the BatchFileAdapter code: rename BatchFileAdapter back to FileAdapter. From now on, FileAdapter will always be called FileAdapter, no matter how many interfaces it has implemented. We can start a new PR which will be handled before this one.

Reply:
@hsluoyz It's correct. When the UpdatableFileAdapter extends the FileAdapter, it will have the methods of the FileAdapter.